pylzy/lzy/env/explorer/base.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from typing import List, Dict
+from typing_extensions import TypeAlias
+
+from lzy.env.base import Deconstructible, WithLogger
+from .search import VarsNamespace
+
+ModulePathsList: TypeAlias = List[str]
+PackagesDict: TypeAlias = Dict[str, str]
+
+
+class BaseExplorer(Deconstructible, WithLogger):
```

Comment:
how many explolers do we have?
if we have only one then let's remove this interface

Reply:
We have only one for now, but I have plans for more explorers in future